includes/Admin/ReviewNotice.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace WeDevs\WeMail\Admin;
+
+/**
+ * Shiw admin notice.
+ */
+class ReviewNotice {
+
+    private static $instance;
+    public $time_based_review;
+    public $day_count;
+    public $campaign_count;
+
+    public static function instance() {         if ( ! isset( self::$instance ) ) {
+            self::$instance = new ReviewNotice();
+            self::$instance->boot();
+	}
+
+        return self::$instance;
+    }
+
+    /**
+     * Boot the notice
+     */
+    private function boot() {
+        add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
+    }
+
+    public function enqueue_assets() {
+        wp_register_style( 'wemail-review-notice-style', WEMAIL_ASSETS . '/css/reviewNotice.css', false, filemtime( WEMAIL_PATH . '/assets/css/reviewNotice.css' ) );
+        wp_register_script( 'wemail-review-notice-script', WEMAIL_ASSETS . '/js/admin-review-notice.js', [ 'jquery' ], filemtime( WEMAIL_PATH . '/assets/js/admin-review-notice.js' ), true );
+
+        wp_enqueue_style( 'wemail-review-notice-style' );
```

Comment:
FYI, You can register scripts and enqueue at once by just using `wp_enqueue_script` It has the same arguments that `wp_register_script` has. If you need to register and enqueue at the same time then you may use the `wp_enqueue_script` instance of the first register and then enqueue.